backend/config/db.js

```diff
@@ -1,7 +1,7 @@
 const mongoose = require('mongoose');
 const config = require('config');
-const db = config.get('mongoURI');
-// const db = config.get('mongoLocal');
+// const db = config.get('mongoURI');
+const db = config.get('mongoLocal');
```

Comment:
This was made by you to use local mongo right? Could you revert the changes made to work with local mongo?

Reply:
This PR still isn't merged, right?
@Pranav2612000 @vasusharma7 

It only solves 'NULL' issue or has any other fixes?

Reply:
It also solves, redirect to login issue for guest.

Reply:
> This was made by you to use local mongo right? Could you revert the changes made to work with local mongo?

yes, I will do